script.js

```diff
@@ -0,0 +1,131 @@
+// Question 1
+console.log ("Hello World!")
+console.log ("Merhaba Dunya!")
+
+// Question 2
+console.log ("I'm awesome")
+
+//Question 3
+  //i
+  let age
+
+  //ii
+  console.log ("The value of age will be:Undefined")
+
+  //iii
+  console.log(age)
+
+  //iv
+  age=34
+
+  //v
+  console.log("The value of age will be:34")
+
+  //vi
+  console.log(age)
+
+//Question 4
+let name = "Kursad"
+
+  //i
+  console.log("the value of my string will be:Kursad")
+
+  //ii
+  console.log(name)
+
+  //iii
+  name = "Begum"
+
+  //iv
+  console.log("the value of my string will be:Kursad")
```

Comment:
Maybe you copied this console.log statement from line 31, but look at line 37 for the new value of `name`.